modules/tracking/samples/lasot_benchmark.py

```diff
@@ -0,0 +1,187 @@
+import numpy as np
+import cv2 as cv
+import sys
+import glob
+import argparse
+
+parser = argparse.ArgumentParser(description="Run benchmark")
+parser.add_argument("--path_to_dataset", type=str,
+                    default="LaSOT", help="Full path to LaSOT folder")
+parser.add_argument("--path_to_results", type=str,
+                    default="results.txt", help="Full path to file with results")
+args = parser.parse_args()
+
+list_of_videos = [None for _ in range(280)]
+file_with_video_names = open(args.path_to_dataset + "/testing_set.txt", "r")
+
+results = open(args.path_to_results, "w")
+
+for i in range(280):
+    list_of_videos[i] = file_with_video_names.readline()
```

Comment:
Can we avoid hardcoded 280?

Reply:
280 is a number of videos in testing part of LaSOT dataset. Whole benchmark is created for working with certain dataset, so I do not think that hardcode is bad here.

Reply:
But we can extract this value from the list of videos

Reply:
Right! I will set it as `len(list_of_videos)` .